flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/SubtaskCheckpointCoordinatorImpl.java

```diff
@@ -61,11 +65,6 @@
 		this.actionExecutor = actionExecutor;
 	}
 
-	@Override
```

Comment:
From the commit "[FLINK-16744][task][refactor] inline CheckpointingOperation", it is unnecessary changes to move the position of this method, if we want to merge this commit separately.